.github/workflows/ci.yml

```diff
@@ -0,0 +1,64 @@
+name: CI
+on: [push, pull_request]
+jobs:
+  test:
+    name: Test
+    strategy:
+      matrix:
+        pg-version: ['10', '11']
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
+
+    - name: Install Redis
+      uses: zhulik/redis-action@v1.0.0
+      with:
+        redis version: '5'
+
+    - name: Install PostgreSQL
+      uses: harmon758/postgresql-action@v1
+      with:
+        postgresql version: ${{ matrix.pg-version }}
+        postgresql db: courier_test
+        postgresql user: courier
+        postgresql password: courier
+
+    - name: Install Linux packages
+      run: sudo apt install -y --no-install-recommends postgresql-client
```

Comment:
if you're not doing the sql loading in a step, then you don't need this

Reply:
right